### DISCUSSION TITLE
Difference b/n this (GitHub) Source Code vs. what UNC Provides Upon Request

### DISCUSSION THREAD

**Discussion by msw17002** [adcirc/adcirc#372] [General]
Is this the developmental version of ADCIRC-SWAN?

I ask since I thought the only way to obtain ADCIRC-SWAN source code was through the University of North Carolina's Coastal Engineering department (upon request and approval).

**Comment by jasonfleming** [comment]
Hey @msw17002 this is the full source code repository, including all release versions. At NOAA's request back in October, the code has been placed under a normal open source license (GNU General Public License). Have at it! :-)

**Reply by msw17002** [reply]
Oh, that's exciting! :D 

Thanks for the info, Jason!
